### DISCUSSION TITLE
Request for Implementation of 'startFrom' Configuration in ngx-highlightjs

### DISCUSSION THREAD

**Discussion by eliasmatheus** [MurhafSousli/ngx-highlightjs#278] [Ideas]
Hello @MurhafSousli,

I hope this message finds you well. I wanted to bring to your attention a potential enhancement for the ngx-highlightjs project, specifically regarding the integration of the `startFrom` configuration from the highlightjs-line-numbers plugin.

As a user of ngx-highlightjs, I've found it immensely valuable in simplifying the use of highlight.js within Angular projects. However, I noticed that the `startFrom` configuration, available in the [highlightjs-line-numbers](https://github.com/wcoder/highlightjs-line-numbers.js?tab=readme-ov-file#startfrom) plugin is not currently implemented in ngx-highlightjs. It allows users to customize the starting line number, which can be incredibly useful in various contexts, including those where precise line numbering is essential.

I believe incorporating the `startFrom` configuration would significantly enhance its functionality and provide users with greater flexibility in displaying code snippets.

Would you consider implementing the `startFrom` configuration in ngx-highlightjs? 

I'm more than willing to provide assistance or collaborate in any way possible to help make this feature a reality. Thank you for considering my request, and I look forward to your response.

Best regards,
Elias Oliveira

**Comment by MurhafSousli** [comment]
Hi, we already have a feature request for this here #274

**Reply by eliasmatheus** [reply]
Hello @MurhafSousli,

Thank you for the response. It was a detail I missed, but it's great to know that issue #274 is handling the 'startFrom' configuration. 

I'm looking forward to seeing these features in action in ngx-highlightjs. They are additions that will certainly make the tool even more powerful and versatile.

Once again, thank you for the update and for all the work you're doing. I look forward to the updates and am available to collaborate if needed.
